proposed/http-client/http-client-meta.md

```diff
@@ -28,6 +28,19 @@ client will allow libraries to be decoupled from an implementation such as Guzzl
 specify the default behaviours. 
 * The purpose is not to be opinionated about the use of middlewares (PSR-15).
 
+## Approaches
+
+### Exceptions
+
+Our the domain exceptions `NetworkException`, `RequestException` and `HttpException` define 
+a contract very similar to eachother. The choosen approach is to not let them extend eachother
+because inheritance does not make sense in the domain model. A `RequestException` is not a 
+`NetworkException`. 
+
+Allowing exception to extend a `RequestAwareException` and/or `ResponseAwareException` interface
+has been discussed but that is a convenience shortcut that one should not take. One should rather
+catch the specific exceptions and handle them accordingly. 
```

Comment:
not sure if this should be part of the PSR itself. or is it common practice to have meta information about the discussion in the PSR itself? it is relevant information, but only for people challenging the standard, not for people that need to know what the standard is.